controlplane/rosa/api/v1beta2/rosacontrolplane_types.go

```diff
@@ -72,6 +72,17 @@ const (
 	Nightly ChannelGroupType = "nightly"
 )
 
+// AutoNodeMode specifies the AutoNode mode
+type AutoNodeMode string
+
+const (
+	// Enabled AutoNode mode
+	Enabled AutoNodeMode = "enabled"
+
+	// Disabled AutoNode mode
+	Disabled AutoNodeMode = "disabled"
+)
```

Comment:
```suggestion
	// AutoNodeModeEnabled sets the AutoNode mode to "Enabled".
	AutoNodeModeEnabled AutoNodeMode = "Enabled"

	// AutoNodeModeDisabled sets the AutoNode mode to "Disabled". 
	AutoNodeModeDisabled AutoNodeMode = "Disabled"
)
```

Reply:
enum values should be with lower case to match ocm-sdk-api. Enable/disable is simpler than AutoNodeModeEnabled/AutoNodeModeDisabled, there is no need be specific in naming dataType is clear.

Reply:
as we discussed, done.